docs/configuration/tracing.md

```diff
@@ -181,3 +181,41 @@ Traefik supports three tracing backends: Jaeger, Zipkin and DataDog.
     #
     prioritySampling = false
 ```
+
+## Instana
+
+```toml
+# Tracing definition
+[tracing]
+  # Backend name used to send tracing data
+  #
+  # Default: "jaeger"
+  #
+  backend = "instana"
+  # Service name used in DataDog backend
```

Comment:
```suggestion
  # Service name used in Instana backend
```

Reply:
@mmatur thank you for the review. i've resolved the suggestion and pushed a new revision.